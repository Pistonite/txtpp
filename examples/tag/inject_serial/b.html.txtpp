<html>
    <head>
        <!-- TXTPP#tag TITLE
        -TXTPP# -->
        <!-- TXTPP#run echo -n "hello tag" 
        -TXTPP# -->
        <title>TITLE</title>
    </head>
    <!-- TXTPP#tag BODY
    -TXTPP# -->
    <!-- TXTPP#include body.html
    -TXTPP# -->
    <body><pre>BODY</pre></body>
</html>