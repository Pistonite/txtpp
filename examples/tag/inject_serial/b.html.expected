<html>
    <head>
        <title>hello tag</title>
    </head>
    <body><pre><span>Praesent semper feugiat nibh s
ed. </span>Enim nec dui nunc mattis enim ut tellus ele
  mentum. Sit amet tellus cras adipiscing. F
aucibus ornare suspendisse sed nisi lacus sed viverra. D
  ictum fusce ut placerat orci. Nibh cras pulvinar mattis n
unc sed blandit libero volutpat sed. Facilisis magna etiam tempor orci eu lobor
  tis elementum. Condimentum lacinia quis vel eros donec ac odio tempor. Ullamcor
per a lacus vestibulum sed arcu non odio. Habitant mor
  bi tristique senectus et netus et <span>malesuada. Posu
ere ac ut consequat semper viverra nam libero ju</span>sto laoreet.</pre></body>
</html>
